cookbooks/bcpc/attributes/default.rb

```diff
@@ -26,7 +26,9 @@
   preseed['late_command'] = 'true'
 
   # All these lines get concatenated, hence the semicolons.
-  preseed['early_command'] = <<-EOM.gsub(/^ {6}/, '')
+  preseed['early_command'] = <<-EOM.gsub(/^ {4}/, '')
+      set -- $(lvm vgscan --rows --noheadings | head -n 1); \\
+      for vg in "$@"; do lvm vgremove -f "$vg"; done; \\
```

Comment:
On our hardware clusters we desire to re-PXE leaving data disks intact to avoid requiring a full HDFS re-balance. Do we have to purge LVM volumes or how could we support that use case?